extensions/amp-sidebar/1.0/toolbar.js

```diff
@@ -103,36 +100,19 @@ export class Toolbar {
     this.toolbarClone_ = this.toolbarDOMElement_.cloneNode(true);
     const targetId = this.toolbarDOMElement_.getAttribute('target');
     // Set the target element to the toolbar clone if it exists.
-    const targetElement =
-    this.win_.document.getElementById(`${targetId}_toolbar`) ||
-    this.win_.document.getElementById(targetId);
-    this.targetElement_ = targetElement || this.createTargetElement_();
-    this.targetElement_.appendChild(this.toolbarClone_);
-    toggle(this.targetElement_, false);
-
-    // Check if the target element was created by us, or already inserted by the user
-    if (!this.targetElement_.parentElement) {
+    const targetElement = this.win_.document.getElementById(targetId);
+    if (targetElement) {
+      this.targetElement_ = targetElement;
       this.toolbarClone_.classList.add(TOOLBAR_ELEMENT_CLASS);
-      const fragment = this.win_
-        .document.createDocumentFragment();
-      fragment.appendChild(this.targetElement_);
-      this.body_.appendChild(fragment);
+      this.targetElement_.appendChild(this.toolbarClone_);
```

Comment:
Do these steps necessary to do in `buildCallback`? It's relatively sensitive for performance. I'd rather do the action once the decisions on placement are being made...